setup.py

```diff
@@ -1,9 +1,20 @@
 from setuptools import setup, find_packages
 
+# We define the version number in restructuredtext_lint/__init__.py
+# Here we can't use "import restructuredtext_lint" to access it
+# as "restructuredtext_lint.__version__" as that would tell us the
+# previously installed version (if any).
```

Comment:
This comment is overly verbose. It should be easily replaced with "Retrieve `__version__` from `__init__.py`"